.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "appium-script-schema"]
 	path = appium-script-schema
 	branch = master
-	url = git@github.com:kobiton/appium-script-schema.git
+	url = https://github.com/kobiton/appium-script-schema.git
```

Comment:
Could you please `rebase` your code base again I remembered that this change I added this [PR](https://github.com/kobiton/appium-script-generator/pull/4/files) last month.

Reply:
What do you mean?